src/plugins/wizard/public/application/utils/wizard_state_validation.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import Ajv from 'ajv';
+
+export const wizardStateValidation = (wizardState) => {
+  const ajv = new Ajv();
+  const vizTypeEnum = ['metric', 'line', 'area', 'histogram'];
+
+  const wizardStateSchema = {
```

Comment:
Can we make a few changes here:
1. Can we move this schema to a schema.json file thats imported here?
2. Can we match the structure of the actual typescript interfaces? e.g. [visualizationState](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/wizard/public/application/utils/state_management/visualization_slice.ts#L11) and styleState (style should actually be an object and not any but lets enforce object)

Reply:
Moved to schema.json and change the schema to match typescript interfaces.